### ISSUE TITLE
"cutoff" needed by chemiscope-input ? 

### ISSUE THREAD

**Issue by gabor1** [lab-cosmo/chemiscope#250]
Hi!

I'm trying to use chemiscope to visualise a PCA map created by ASAP, using per-atom environments. I created the attached file with ASAP, it's an XYZ because the direct chemiscope output generation is broken. When I use the following command to generate a json, I get an error about a cutoff. There is no argument to chemiscope-input where I could specify a cutoff (I am not sure why a cutoff is required, since I already have the PCA coordinates in the input file, all I am trying to do is get chemiscope to put the environments on a map... ). I tried specifying a cutoff in the info line of the xyz, but that didn't work either. 

```
Gabors-MacBook-Pro-2:Downloads gabor$ chemiscope-input ASAP-lowD-map.xyz --only-atoms -o tmp.json
Traceback (most recent call last):
  File "/Users/gabor/Library/Python/3.8/bin/chemiscope-input", line 8, in <module>
    sys.exit(main())
  File "/Users/gabor/Library/Python/3.8/lib/python/site-packages/chemiscope/main.py", line 73, in main
    if args.only_atoms and args.cutoff is None:
AttributeError: 'Namespace' object has no attribute 'cutoff'
Gabors-MacBook-Pro-2:Downloads gabor$ 
```

[ASAP-lowD-map.xyz.gz](https://github.com/lab-cosmo/chemiscope/files/9724727/ASAP-lowD-map.xyz.gz)

thanks!

Gabor

**Comment by gabor1** [comment]
Could you put back the cutoff argument then? The problem is that asap chromoscope export is just broken for per-atom descriptors

**Comment by gabor1** [comment]
@ceriottm scripy doesn't deal with the cutoff as far as I can tell

**Comment by ceriottm** [comment]
`chemiscope.all_atomic_environments(frames,cutoff=6)` generates a list of environments with a cutoff of six angstrom.

**Comment by gabor1** [comment]
Asap just stops with a syntax error. The problem is not on the chemiscope side

**Comment by ceriottm** [comment]
I am closing this since we merged #248 that streamlines the definition of the properties. Perhaps one could create an issue on the ASAP side so that the interface is brought up to the current syntax, and then the version pinned so that future changes on the chemiscope side won't break anything there.
